liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/AbstractLibertySupport.java

```diff
@@ -30,32 +32,19 @@
 public abstract class AbstractLibertySupport extends MojoSupport {
     /**
      * Maven Project
-     * 
-     * @parameter expression="${project}"
-     * @required
-     * @readonly
      */
+    @Parameter(defaultValue = "${project}",required = true,readonly = true)
     protected MavenProject project = null;
-
-    /**
-     * @parameter expression="${localRepository}"
-     * @readonly
-     * @required
-     */
+    @Parameter(defaultValue = "${localRepository}",required = true,readonly = true)
```

Comment:
coding style: need a space after comma and recommend a new line between after line 37  
Same comment for other @Parameter annotation in this class